fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -190,6 +191,13 @@ class FullyShardedDataParallel(nn.Module):
             module to be wrapped with FSDP.
         process_group (Optional):
             process group for sharding
+        process_group_reduce_scatter (Optional):
+            process group for reduce scatter
+            it defaults to None. A seperate process group is initialized and assigned to the reduce_scatter operation. And the
```

Comment:
instead of default to None, why not default to ProcessGroupName.reduce_scatter? That way, the type of this is Union[str, PG] and it not an optional anymore.

Reply:
Good idea.